querybook/server/app/server.py

```diff
@@ -34,4 +34,4 @@ def get_health_check():
 @flask_app.route("/<path:ignore>")
 @limiter.exempt
 def main(ignore=None):
-    return send_file(WEBAPP_INDEX_PATH, mimetype="text/html")
+    return redirect("/db-explorer/", 302)
```

Comment:
Adding redirect to the frontend, since the backend image no longer has the frontend files.